### ISSUE TITLE
keyboard navigation doesn't work initially with in-product release notes

### ISSUE THREAD

**Issue by gregvanl** [Microsoft/vscode#13264]
- VSCode Version:1.6.0 insiders
- OS Version:Windows 10

Steps to Reproduce:
1. Follow test plan to enable in product release notes in insiders https://github.com/Microsoft/vscode/issues/12520
2. Help > Release Notes
3. Try up/down/pageup/pagedown

The editor is not scrolled.  Only if you click on the editor, does keyboard navigation start to work.
This is also the case if you go to another editor and then click the tabbed header or use Quick Open to activate the release notes.

**Comment by joaomoreno** [comment]
Cherry picked to `release/1.6`

**Comment by joaomoreno** [comment]
Rolled back cherry pick. Removing it from candidates. Can't merge this in without merging #12982 which wasn't properly smoke tested.

**Comment by chrmarti** [comment]
Removing candidate label and updating milestone to October.
